rest/reference.go

```diff
@@ -25,6 +25,10 @@ const (
 	ListConditionsPath = "/v3/reference/conditions"
 
 	GetExchangesPath = "/v3/reference/exchanges"
+
+	GetOptionsContractPath = "/v3/reference/options/contracts/{options_ticker}"
+
```

Comment:
I've been grouping these paths intentionally so we should get rid of this extra space here